### PR TITLE
Bug 1269416 - Highlight inside some tags

### PR DESCRIPTION
**Pull request by KWierso**
This change is [Reviewable](https://reviewable.io/reviews/mozilla/treeherder/1685)

### CODE REVIEW

File: ui/js/filters.js [hunk 1]
```diff
@@ -143,8 +143,10 @@ treeherder.filter('highlightCommonTerms', function() {
         angular.forEach(tokens, function(elem) {
             if (elem.length > 0) {
                 input = input.replace(new RegExp("(^|\\W)(" + elem + ")($|\\W)", "gi"), function(match, prefix, token, suffix, index, str) {
-                    if (inTag(str, index, "<", ">") || inTag(str, index, "&", ";")){
+                    if (inTag(str, index, "<", ">")) {
                         return match;
+                    } else if (inTag(str, index, "&", ";")) {
+                        return prefix + "<strong>" + token + "</strong>" + suffix;
```
Comment (KWierso) [inline ui/js/filters.js]: Wondering if this should only do the <strong>ifying when the "tag" is '#39' or 'quot'...

**Comment by jgraham** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[ui/js/filters.js, line 149 \[r1\]](https://reviewable.io:443/reviews/mozilla/treeherder/1685#-KM_gmj8kuoPqyI_fn_R-r1-149:-KMxuNNbonzMGMFTSriy:-2114491672) ([raw file](https://github.com/mozilla/treeherder/blob/6f1997b6531d37b3bd92c26e97e0f5813716ffa1/ui/js/filters.js#L149)):*
<details><summary><i>Previously, KWierso wrote…</i></summary>
> Wondering if this should only do the strong-ifying when the "tag" is '#39' or 'quot'...
</details>
This entire method seems like such a hack. It seems like a more reasonable approach would be to just write the escaped string into the DOM and then do the per-term highlighting in javascript where we would have a real HTML parser to use and none of the problems that `inTag` half solves.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/treeherder/1685)*

**Comment by KWierso** [comment]
*[ui/js/filters.js, line 149 \[r1\]](https://reviewable.io:443/reviews/mozilla/treeherder/1685#-KM_gmj8kuoPqyI_fn_R-r1-149:-KN0toFZNNFZNVvkEsdU:-1487229163) ([raw file](https://github.com/mozilla/treeherder/blob/6f1997b6531d37b3bd92c26e97e0f5813716ffa1/ui/js/filters.js#L149)):*
<details><summary><i>Previously, jgraham wrote…</i></summary>
> This entire method seems like such a hack. It seems like a more reasonable approach would be to just write the escaped string into the DOM and then do the per-term highlighting in javascript where we would have a real HTML parser to use and none of the problems that `inTag` half solves.
</details>
Hmm, wrap each "word" in the summary as a separate html tag, then we can apply a class to the ones we want to make bold?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/treeherder/1685)*

**Comment by KWierso** [comment]
We went a different way in PR 1818.

**Pull request closed without merging**